src/services/Queries.js

```diff
@@ -296,5 +317,97 @@ class Queries {
 
     return false;
   }
+
+  /**
+   * find trip function
+   * @param {String} table table name
+   * @param {Object} tripId trip id
+   * @returns { Object } result
+   */
+  static async getTripRequestByTripId(table, tripId) {
+    const data = await table.findAll({
+      where: {
+        tripId,
+      }
+    });
+    return data;
+  }
+
+  /**
+   *
+   * Function to update the status to approved or reject
+   * @param {String} table the name of the table to updated
+   * @param {string} tripId the id of the selected trip
+   * @param {Object} data it gets the data from req.body
+   * @returns {object} updated data
+   */
+  static async updateTripRequest(table, tripId, data) {
+    try {
+      const updatedRequest = await table.update(
+        data,
+        { where: { tripId } }
+      );
+      return updatedRequest;
+    } catch (error) {
+      return error;
+    }
+  }
+
+  /**
+   *
+   * Function to update the status to approved or reject
+   * @param {String} table the name of the table to updated
+   * @param {string} tripId the id of the selected trip
+   * @param {Object} data it gets the data from req.body
+   * @returns {object} updated data
+   */
+  static async updateMultiCityTripRequest(table, tripId, data) {
+    try {
+      const updatedRequest = await table.update(
+        data,
+        { where: { id: tripId } }
+      );
+      return updatedRequest;
+    } catch (error) {
+      return error;
+    }
+  }
+
+  /** Function to update the status to approved or reject
+   *
+   * @param {object} table to be updating in
+   * @param {integer} tripId trip id to be found and then replace the status
+   * @returns {object} updated data
+   */
+  static async updateRequestStatus(table, tripId) {
+    try {
+      const updatedRequest = await table.update(
+
+
+        { status: 'pending' },
+        { where: { tripId } }
+      );
+      return updatedRequest;
+    } catch (error) {
+      return error;
+    }
+  }
+
+  /**
+   *
+   * This method will be user to delete the selected trip in multi city
+   * @param {String} table the name of the table to updated
+   * @param {string} tripId the id of the selected trip
+   * @param {Object} data it gets the data from req.body
+   * @returns {object} updated data
+   */
+  static async deleteMultiCityTripRequest(table, tripId) {
```

Comment:
@mujohn26 is this function about deleting only a multi-city request or multi-city trip? I think there should be generic functions to delete trip-request and trip given IDs